### ISSUE TITLE
LORA-RA02 module

### ISSUE THREAD

**Issue by htarahi** [sandeepmistry/arduino-LoRa#340]
Hi every one
I have 2 arduino with 2 lora-ra02 modules that works in 433 MHz frequency.
I upload the code but it do not receive any thing !
both sides send packet but receive nothing !
what can be the problem ?
ofcourse some times sending stop working with no error.
![Lora](https://user-images.githubusercontent.com/61028817/75089499-b20a3b80-556e-11ea-8ed7-a892a71d6ecf.jpg)

**Comment by IoTThinks** [comment]
Sending means nothing.
Only receiving counts the success.

Check the LoRa pin mapping, SetPins() and LoRa settings on both side.

**Comment by htarahi** [comment]
This time I use the simplest code ...Lora sender & lora Receiver
again I have no receive !
I use arduino uno with lora 02 in both sides.
arduino board's Power is from USB, connected to PC.
arduino provides lora module power.
frequency is 433 (433E6) with pins connection as below.
![pins](https://user-images.githubusercontent.com/61028817/75090934-2fd54380-557d-11ea-9161-db8ca7aba430.jpg)

**Comment by htarahi** [comment]
no idea yet ??!

**Comment by 04-Hax** [comment]
Same Issue. Did you find the solution?

**Comment by Kongduino** [comment]
Do you have an SDR ? It would allow you to verify that the packets are actually sent, and sent properly. I had an issue with a device once – the SDR showed it was sending, but at BW 250 KHz instead of the requested 125 KHz, or something similar. Which explained why other devices couldn't receive...

**Comment by 04-Hax** [comment]
Yes I have already cross checked it on Freq 433Mhz , and at the receiver side also it's tuned at 433Mhz

**Comment by 04-Hax** [comment]
I'm having Rx only , i check that LoRa is sending packets on frequency 433MHz but receiver side LoRa is not capturing anything.

**Comment by IoTThinks** [comment]
Show your code, your wiring and real photo of your boards.

**Comment by 04-Hax** [comment]
I'm using example code , with 2 Arduino Uno r3 attached with Ai Thinker Ra-02.
![Screenshot_20210530-120817_GitHub.png](https://user-images.githubusercontent.com/38910652/120094730-c78dee00-c13f-11eb-8bee-2e3491770b26.png)

**Comment by IoTThinks** [comment]
@04-Hax With your own eyes on your board and your code, you can tell what is wrong.
How could anyone tell without seeing the code and real photo?

Wow, example codes. Which example codes? :D 
Must be a fortune teller to know.

Nevermind, let other people to help.

**Comment by 04-Hax** [comment]
Receiver Example.
At first it was giving me error , LoRa starting failed.Then I fixed it by making changes in cpp file.
Now everything is working good but the packets which are transmitted by LoRa is not getting received by another module.
